jax/experimental/jax2tf/tests/call_tf_test.py

```diff
@@ -0,0 +1,284 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for call_tf."""
+
+from typing import Callable, Dict, Tuple
+import unittest
+
+from absl.testing import absltest
+from absl.testing import parameterized
+
+import jax
+from jax import lax
+from jax import numpy as jnp
+from jax import test_util as jtu
+from jax.config import config
+from jax.experimental import jax2tf
+
+import numpy as np
+
+try:
+  import tensorflow as tf  # type: ignore[import]
+except ImportError:
+  tf = None
+
+config.parse_flags_with_absl()
+
+
+def _maybe_jit(with_jit: bool, func: Callable) -> Callable:
+  if with_jit:
+    return jax.jit(func)
+  else:
+    return func
+
+
+parameterized_jit = parameterized.named_parameters(
+    dict(testcase_name="_jit" if with_jit else "", with_jit=with_jit)
+    for with_jit in [True, False])
+
+
+class CallTfTest(jtu.JaxTestCase):
+
+  def setUp(self):
+    if tf is None:
+      raise unittest.SkipTest("Test requires tensorflow")
+    super().setUp()
+
+  @parameterized_jit
+  def test_eval(self, with_jit=True):
```

Comment:
nit: Why provide a default value for with_jit?

Reply:
There is a reason. When I run the test in my IDE, I comment out the @parameterized_jit, and then with one click I can run `test_eval`.